lib/gslab_model/test/test.ExampleModel.R

```diff
@@ -0,0 +1,27 @@
+source("ExampleModel.R")
+
+test.ExampleModel.initialize <- function() {
+    a <- ExampleModel("y", c("x1", "x2"), suffix = "_c", include_constant = 1, startparam = c(0, 1, 2))
+    checkEquals(c(a$paramlist, a$lhslist, a$rhslist, a$nparam, a$include_constant, a$startparam),
+                c(c("x1_c", "x2_c", "constant"), "y", c("x1","x2"), 3, 1, c(0, 1, 2)))
+    b <- ExampleModel(c("y1", "y2"), "z", include_constant = 0)
+    checkEquals(c(b$paramlist, b$lhslist, b$rhslist, b$nparam, b$include_constant, b$startparam),
+                c("z_coeff", c("y1", "y2"), "z", 1, 0, 0))
+    checkException(ExampleModel("y", "x", include_constant = 0, startparam = c(2, 2)), silent = TRUE)
+}
+
+test.ExampleModel.XBeta <- function() {
```

Comment:
Maybe check function when not every RHS variable in ModelData is included in the X matrix?